gallery/src/styles/Nivo/NivoPage.tsx

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { NxTable, NxCode, NxP, NxH3, NxTextLink } from '@sonatype/react-shared-components';
+
+import { GalleryDescriptionTile, GalleryExampleTile } from '../../gallery-components/GalleryTiles';
+
+import NivoLineExample from './NivoLineExample';
+import NivoBarExample from './NivoBarExample';
+import NivoPieExample from './NivoPieExample';
+
+const NivoLineExampleCode = require('./NivoLineExample?raw');
+const NivoBarExampleCode = require('./NivoBarExample?raw');
+const NivoPieExampleCode = require('./NivoPieExample?raw');
+
+const NivoPage = () =>
+  <>
+    <GalleryDescriptionTile>
+      <NxP>
+        <NxTextLink external href="https://nivo.rocks/components/">Nivo</NxTextLink> charts are
+        recommended for data visualization.
+      </NxP>
+      <NxH3>Accessibility</NxH3>
+      <NxP>
+        The color array show below is recommended for use in the Nivo charts.
+      </NxP>
+      <NxTable>
+        <NxTable.Head>
+          <NxTable.Row>
+            <NxTable.Cell>Priority </NxTable.Cell>
+            <NxTable.Cell>Color</NxTable.Cell>
+          </NxTable.Row>
+        </NxTable.Head>
+        <NxTable.Body>
+          <NxTable.Row>
+            <NxTable.Cell>First</NxTable.Cell>
+            <NxTable.Cell><NxCode>nx-teal-40</NxCode></NxTable.Cell>
```

Comment:
Consider pulling `ColorDocRow` out of `CssVariablesPage` for usage here.  In any event it would be good if the rows actually included an example of what the color looks like.

Reply:
> Consider pulling `ColorDocRow` out of `CssVariablesPage` for usage here. In any event it would be good if the rows actually included an example of what the color looks like.

Ended up removing the table: f7c3a5a83d6ce69959fdfaab0c8517f80022adef